src/popup/components/PanelItem.vue

```diff
@@ -35,6 +35,13 @@
         />
       </slot>
     </div>
+    <template v-if="$slots['bottom-right-icon']">
+      <div class="panel-item-bottom-right">
```

Comment:
```suggestion
    <div
      v-if="$slots['bottom-right-icon']"
      class="panel-item-bottom-right"
    >
```
No need to use template here.